### PR TITLE
Add utility function and tests

### PR DESCRIPTION
**Pull request by kangarang**

### CODE REVIEW

**Comment by kangarang** [comment]
Hi @skmgoldin , I'm all done editing this PR. Thanks for the code review!

File: test/plcrVoting.js [hunk 1]
```diff
@@ -156,6 +156,21 @@ contract('PLCRVoting', (accounts) => {
...
+
+    it('should throw an error when attempting to rescue tokens from a non-existant poll',
+      async () => {
+        const plcr = await utils.getPLCRInstance();
+        const options = defaultOptions();
+        options.actor = bob;
+
+        try {
+          await utils.as(bob, plcr.rescueTokens, '667');
+          // assert(false,
```
Comment (skmgoldin) [inline test/plcrVoting.js]: Uncomment this and let this test fail where it should fail.

**Comment by skmgoldin** [comment]
Once you make that fix, could you start a `develop` branch and remake this PR to `develop`? This will make our tests fail and give us a red badge on master. Which we deserve. But I don't want it.

**Comment by kangarang** [comment]
Sure thing. Please find #8.

**Pull request closed without merging**